music21/voiceLeading.py

```diff
@@ -605,43 +654,34 @@ def parallelInterval(self, thisInterval):
         Returns boolean.
         '''
 
-        if not (self.parallelMotion() or self.antiParallelMotion()):
-            return False
-
-        if isinstance(thisInterval, str):
-            thisInterval = interval.Interval(thisInterval)
+        return (self.parallelMotion(
+                    requiredInterval=thisInterval,
+                    allowOctaveDisplacement=True
+                    )
+                or self.antiParallelMotion())
```

Comment:
This should probably also pass requiredInterval to antiParallelMotion() as well, since it `thisInterval` could be P5 and antiParallelMotion could be `P8`

Reply:
Yes! antiParallelMotion() currently doesn't have a requiredInterval flag, but this can be added.